docs/MSALBrowserAuthenticationProvider.md

```diff
@@ -0,0 +1,64 @@
+#### Creating an instance of MSALBrowserAuthenticationProvider for a browser application
+
+**Note**: The `MSALBrowserAuthenticationProvider` is introduced in version 3.0.0 of Microsoft Graph Client Library
+
+###### Links for more information -
+
+-   [npm - @azure/msal-browser](https://www.npmjs.com/package/@azure/msal-browser)
+-   [github - @azure/msal-browser](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/README.md)
+
+Steps to use `MSALBrowserAuthenticationProvider`;
+
+1.  Using npm: `npm install @azure/msal-browser`
+
+    Using html:
+
+    ```html
+    <!--Using script tag to include the bundled file or the CDN source-->
+    <script type="text/javascript" src="https://alcdn.msauth.net/browser/2.15.0/js/msal-browser.min.js"></script>
```

Comment:
use of single tags? 
integrity and cross origin attributes?

Reply:
Added reference to the browser cdn usage for more information. The sample uses the basic usage of CDN for the source.